deploy.tf

```diff
@@ -381,10 +374,154 @@ EOF
 
 ###############################################################################
 #
-# Make config file and export variables for kubectl
+# etcd host firewall
+#
+###############################################################################
+
+resource "digitalocean_firewall" "k8s_etcd" {
+  name = "${var.prefix}k8s-etcd-fw"
+
+  droplet_ids = ["${digitalocean_droplet.k8s_etcd.id}"]
+
+  inbound_rule = [
+    {
+      protocol           = "tcp"
+      port_range         = "22"
```

Comment:
Curious why we need ssh access from master to etcd?

Reply:
This way etcd droplet is available for debugging over ssh without opening port 22 to the world wide web. Unless I'm mistaken 22 is also open from master to worker nodes. This way if you want to ssh to etc or worker you can use `ProxyJump` in your ssh_config to proxy through master.